_posts/2018-04-03-bug-bbq-how-to-be-involved.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+permalink: 2018-04-03-bug-bbq-involved
+subheadline: "What can I do during the Bug BBQ?"
+title: "How to be involved in the Bug BBQ?"
+teaser: "A short guide on how to contribute and be involved during The Carpentries Bug BBQ."
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["François Michonneau"]
+---
+
+
+## I want to contribute to the lessons during the Bug BBQ!
+
+We created a guide that outlines [how to contribute](https://hackmd.io/s/r1EDctqcG#) to the Bug BBQ. There are many ways to get involved, the issues to address are as varied as the skills they require, and the time it takes to fix them.
+
+As indicated in the guide, we strongly recommend that you leave a comment on the issue you are going to address before starting to work on it. This ensures that no work gets duplicated.
+
+We will use Slack to communicate and coordinate. Feel free to ask questions in the `#general` channel or in the channel associated with the lesson you are working on. 
```

Comment:
Add link to Bug BBQ website here so the list of Slack channels is easily available.